Core/src/main/java/com/plotsquared/core/command/FlagCommand.java

```diff
@@ -145,9 +146,16 @@ private static boolean checkPermValue(
             } catch (final Exception e) {
                 return false;
             }
-            return true;
+            return true;//
+        }
+        boolean result;
+        String basePerm = Permission.PERMISSION_SET_FLAG_KEY.format(key.toLowerCase());
+        if (flag.isValuedPermission()) {
```

Comment:
`valued` or `keyed`? Sounds confusing

Reply:
See #3201

Reply:
It checks hasKeyedPermission inside of that if. The other PR doesn't clear that up for me

Reply:
The comment is better suited to that PR really I mean^